src/main/java/org/broadinstitute/hellbender/tools/copynumber/GermlineCNVCaller.java

```diff
@@ -0,0 +1,408 @@
+ *      and interval-related arguments will be ignored.</p>
+ *
+ *      <p>The calls subdirectory contains one subdirectory for each sample, listing various sample-specific
+ *      quantities such as the probability of various copy-number states for each interval, the GC curve,
+ *      sample-specific unexplained variance, read depth, and loadings of various coverage bias factors.</p></dd>
+ *
+ *     <dt>CASE run-mode</dt>
+ *     <dd><p>The tool can be run in the CASE mode by using the argument <code>--run-mode CASE</code>. The path to a
+ *      previously obtained coverage model parameter bundle must be provided via <code>--model &lt;previous_model_path&gt;</code>.
+ *      The range of genomic intervals is set to the range used for creating the parameter bundle and interval-related
+ *      arguments will be ignored. The output of the CASE run-mode is only the "-calls" subdirectory.</p>
+ * </dl>
+ *
+ * <h3>Important Remarks</h3>
+ * <dl>
+ *     <dt>Choice of hyperparameters</dt>
+ *     <dd><p>The quality of inferred coverage model parameters and germline CNV events is sensitive to the choice of
+ * model hyperparameters, such as the prior probability of alternative CNV states, active regions, the coherence
+ * length of CNV events and active/silent domains, and the typical scale of interval- and sample-specific unexplained
+ * variance. These hyperparameters are not universal and must be properly tuned for each sequencing protocol.</p></dd>
+ *
+ *     <dt>Running {@link GermlineCNVCaller} on a subset of intervals</dt>
+ *     <dd><p>As mentioned earlier, it may be necessary to run the tool over a subset of all intervals depending on
+ * available memory. The number of intervals must be large enough to include a contextually diverse set of regions
+ * for reliable inference of the GC bias curve, as well as other bias factors. For WES and WGS, we recommend no less
+ * than 10000 consecutive intervals spanning at least 10 - 50 mb.</p></dd>
+ * </dl>
+ *
+ * <h3>Examples</h3>
+ *
+ * <p>COHORT run-mode:</p>
+ * <pre>
+ * gatk-launch --javaOptions "-Xmx4g" GermlineCNVCaller \
+ *   -L intervals.interval_list \
+ *   --input normal_1.counts.hdf5 \
+ *   --input normal_2.counts.hdf5 \
+ *   ... \
+ *   --output output_dir \
+ *   --output-prefix normal_cohort_run
+ * </pre>
+ *
+ * <p>CASE run-mode:</p>
+ * <pre>
+ * gatk-launch --javaOptions "-Xmx4g" GermlineCNVCaller \
+ *   -L intervals.interval_list \
+ *   --model previous_model_path \
+ *   --input normal_1.counts.hdf5 \
+ *   ... \
+ *   --output output_dir \
+ *   --output-prefix normal_case_run
+ * </pre>
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+@CommandLineProgramProperties(
+        summary = "Denoise and call copy-number variants in germline samples given their fragment counts and " +
```

Comment:
`fragment counts` -> `counts` here and below.

Reply:
Done.